modules/timeline.xql

```diff
@@ -61,11 +71,17 @@ let $output :=  if(exists($entry/atom:category)) then
             <a href="{$path}/{$entry/wiki:id/string()}" class="exist-read-more">Read more</a>
             <span class="exist-date">{
                 if ($age < xdt:dayTimeDuration("PT1H")) then
-                    minutes-from-duration($age) || " minutes ago"
+                    let $minutes := if (minutes-from-duration($age) = 1) then ' minute' else ' minutes' return
```

Comment:
Cool, I see you enhanced the script to correctly use the singular form.
It would make sense to store `minutes-from-duration($age)` in a variable and use it in both occasions here.
Same applies to hours and days.